tutorials/graphrag/GraphRAGDemo.ipynb

```diff
@@ -200,11 +200,15 @@
     "## Comparing Document Search Methods\n",
     "\n",
     "TigerGraph GraphRAG provides multiple methods to search documents in the graph. The methods are:\n",
-    "- **Hybrid Search**: This method uses a combination of vector search and graph traversal to find the most relevant information to the query. It uses the selected algorithm to search the embeddings of documents, document chunks, entities, and relationships. These results serve as the starting point for the graph traversal. The graph traversal is used to find the most relevant information to the query.\n",
+    "- **Similarity Search**: This is the baseline, traditional vector search approach. It compares the user query against the embeddings of document chunks, and returns the closest top-k matches. While efficient, this method is limited to semantic similarity and does not capture additional context or structural connections.\n",
     "\n",
-    "- **Similarity Search**: This method uses the selected algorithm to search the embeddings of one of the document, document chunk, entity, or relationship vector indices. It returns the most relevant information to the query based on the embeddings. This method is what you would expect from a traditional vector RAG solution.\n",
+    "- **Contextual Search**: This method enhances vector search by incorporating sibling relationships (e.g., IS_AFTER) between document chunks. Starting from the top-k most relevant chunk, it expands to include surrounding chunks, ensuring that context spanning across multiple paragraphs or sections is retrieved. This provides a more complete answer when key information is distributed across adjacent text.\n",
```

Comment:
"This method enhances vector search by incorporating sibling relationships" -> This method enhances vector search by incorporating graph traversal via sibling relationships"